.github/workflows/build-scheds.yml

```diff
@@ -51,16 +47,21 @@ jobs:
       # Install virtme-ng
       - run: pip install virtme-ng
 
-      # Download a sched-ext enabled kernel
-      - run: apt download linux-image-unsigned-6.7.0-4-generic linux-modules-6.7.0-4-generic
-      - run: mkdir -p kernel
-      - run: for f in *.deb; do dpkg -x $f kernel; done
+      # Get the latest sched-ext enabled kernel directly from the git repository
+      - run: git clone https://github.com/sched-ext/sched_ext.git linux
```

Comment:
The dev branch is `sched_ext`. One worry is that when we make breaking changes in the kernel, this would immediately break CI for scx repo. Would it be possible to make it try `sched_ext-ci` first and then if that's not present try `sched_ext` so that we have a way to lag a bit when necessary?

Reply:
Alternatively, we could have a release source tarball for CI that we get and untar here. That would also speed up CI as we wouldn't have to clone the entire repo.

Reply:
Then we could just go with `git clone --single-branch -b sched-ext --depth 1`  and it would be like fetching a tarball.

Reply:
...and git ls-remote or similar to check if sched-ext-ci or sched-ext exist.